config.py

```diff
@@ -24,7 +24,7 @@ class Config(object):
     DM_DATA_API_URL = None
     DM_DATA_API_AUTH_TOKEN = None
     DM_NOTIFY_API_KEY = None
-    DM_CLARIFICATION_QUESTION_EMAIL = 'digitalmarketplace@mailinator.com'
+    DM_CLARIFICATION_QUESTION_EMAIL = 'clarification-questions@example.gov.uk'
```

Comment:
Do we want `example.gov.uk` or `example.com`?

Reply:
I don't think it actually makes a difference here, but I went `gov` simply because it's mimicking what will presumably be a CCS address IRL.